external-crates/move/crates/move-vm-integration-tests/src/tests/loader_tests.rs

```diff
@@ -62,6 +62,12 @@ struct RelinkingStore {
     type_origin: BTreeMap<(ModuleId, Identifier), ModuleId>,
 }
 
+fn parse_type_tag(s: &str) -> anyhow::Result<TypeTag> {
+    move_command_line_common::types::ParsedType::parse(s)
+        .unwrap()
+        .into_type_tag(&|_| None)
+}
```

Comment:
It just seems a bit clunky to need to define these "parse_type_tag" functions in each crate that wants to do a parse form string -> TypeTag